packages/dds/tree/src/events/events.ts

```diff
@@ -83,7 +82,10 @@ export type TransformEvents<E extends Events<E>, Target extends IEvent = IEvent>
  *   error: (errorCode: number) => void;
  * }>
  * ```
- * @beta
+ * @privateRemarks
+ * {@link createEmitter} can help implement this interface via delegation.
```

Comment:
Looks like you can get away with {link} references to internal types from the privateRemarks section.